client-registry-jempi/docker-compose.combined.yml

```diff
@@ -105,28 +102,8 @@ services:
       kafka:
       default:
 
-  jempi-bootstrapper:
-    image: jembi/jempi-bootstrapper:${JEMPI_BOOTSTRAPPER_IMAGE_TAG}
-    environment:
-      POSTGRESQL_IP: ${JEMPI_REPMGR_PRIMARY_HOST}
-      POSTGRESQL_PORT: 5432
-      POSTGRESQL_USER: ${POSTGRESQL_USERNAME}
-      POSTGRESQL_PASSWORD: ${POSTGRESQL_PASSWORD}
-      POSTGRESQL_DATABASE: ${POSTGRESQL_DATABASE}
-      POSTGRESQL_USERS_DB: ${POSTGRESQL_USERS_DB}
-      POSTGRESQL_NOTIFICATIONS_DB: ${POSTGRESQL_NOTIFICATIONS_DB}
-      POSTGRESQL_AUDIT_DB: ${POSTGRESQL_AUDIT_DB}
-      POSTGRESQL_KC_TEST_DB: ${POSTGRESQL_KC_TEST_DB}
-      KAFKA_BOOTSTRAP_SERVERS: ${KAFKA_HOSTS}
-      KAFKA_APPLICATION_ID: ${KAFKA_APPLICATION_ID_BOOTSTRAPPER}
-      DGRAPH_HOSTS: ${DGRAPH_HOSTS}
-      DGRAPH_PORTS: ${DGRAPH_PORTS}
-    networks:
-      kafka:
-      default:
-
```

Comment:
Any reason why we have this service omitted?

Reply:
mistakenly remove, but I have re-added the code